### PR TITLE
work well with other testing frameworks in test-kitchen

### PR DESCRIPTION
**Pull request by chris-rock**

### CODE REVIEW

**Comment by chris-rock** [comment]
fixes #11

**Comment by chris-rock** [comment]
We should add detection if the path contains directories like `serverspec`, `busser` and fallback into legacy mode only then. By default the recommended way is to use `test/integration/%suitename%`

File: lib/kitchen/verifier/inspec.rb [hunk 1]
```diff
@@ -132,7 +150,6 @@ def runner_options_for_ssh(config_data)
         }
         opts['key_files'] = kitchen[:keys] unless kitchen[:keys].nil?
         opts['password'] = kitchen[:password] unless kitchen[:password].nil?
-
         opts
```
Comment (srenatus) [inline lib/kitchen/verifier/inspec.rb]: If you want to save space here, why not also drop the `opts`? It's the return value of the assignment anyways. And we might as well drop that assignment and just return the hash. ;)

Comment (srenatus) [inline lib/kitchen/verifier/inspec.rb]: Anyway, LGTM :+1:

**Pull request merged**